test/performance/README.md

```diff
@@ -1,6 +1,6 @@
 ## Workload Benchmark for KubeStellar
 
-*Pre-requisite*: in order to follow the instructions below, you must have an environment with KubeStellar installed; see [KubeStellar getting started](https://docs.kubestellar.io/release-0.23.1/direct/get-started/). Alternatively, you can also use KubeStellar e2e script [run-test.sh](https://github.com/kubestellar/kubestellar/blob/main/test/e2e/run-test.sh) to setup an environment.
+*Pre-requisite*: in order to follow the instructions below, you must have [python3](https://www.python.org/downloads/) and a environment with KubeStellar installed; see [KubeStellar getting started](https://docs.kubestellar.io/release-0.23.1/direct/get-started/). Alternatively, you can also use KubeStellar e2e script [run-test.sh](https://github.com/kubestellar/kubestellar/blob/main/test/e2e/run-test.sh) to setup an environment.
 
 To generate the sample workload for KubeStellar performance experiments proceed as following:
 
```

Comment:
I tried testing, and ran into trouble pretty quickly. Here is the tail of it:

```shell
mspreitz@mjs13 kubestellar % cd test/performance 
mspreitz@mjs13 performance % ./setup-clusterloader2.sh 
+ set -e
+ env=kind
+ '[' '' == --env ']'
+ :
+ : -------------------------------------------------------------------------
+ : 'Configuring clusterloader2 to generate KS performance workloads'
+++ dirname -- ./setup-clusterloader2.sh
++ cd -- .
++ pwd
+ SCRIPT_DIR=/Users/mspreitz/go/src/github.com/kubestellar/kubestellar/test/performance
+ cp /Users/mspreitz/go/src/github.com/kubestellar/kubestellar/test/performance/cleanup.sh /Users/mspreitz/go/src/k8s.io/perf-tests/
+ cp /Users/mspreitz/go/src/github.com/kubestellar/kubestellar/test/performance/workloads/modules/cluster-density-k8s.yaml /Users/mspreitz/go/src/github.com/kubestellar/kubestellar/test/performance/workloads/modules/cluster-density-ocp.yaml /Users/mspreitz/go/src/github.com/kubestellar/kubestellar/test/performance/workloads/modules/performance-test-module.yaml /Users/mspreitz/go/src/k8s.io/perf-tests/testing/load/modules
cp: /Users/mspreitz/go/src/k8s.io/perf-tests/testing/load/modules is not a directory
mspreitz@mjs13 performance % 
mspreitz@mjs13 performance % echo $?
1
mspreitz@mjs13 performance % ls $CL2_DIR
CONTRIBUTING.md				_hook					network
LICENSE					adhoc					perfdash
Makefile				application-benchmarks			run-e2e-with-prometheus-fw-rule.sh
OWNERS					benchmark				run-e2e.sh
OWNERS_ALIASES				cleanup.sh				slo-monitor
README.md				clusterloader2				util-images
RELEASE.md				code-of-conduct.md			verify
SECURITY.md				dns
SECURITY_CONTACTS			golang
mspreitz@mjs13 performance % 
```

Reply:
https://github.com/kubernetes/perf-tests/tree/release-1.31 shows no directory named "testing".

Reply:
@MikeSpreitzer, it seems that you didn't set your CL2_DIR correctly. You should set it as following: 
```
export CL2_DIR=$HOME/perf-tests/clusterloader2
```

Reply:
this command `ls $CL2_DIR` should show the contents of this directory instead:  `path/perf-tests/clusterloader2`

Reply:
With a little `mkdir -p`, I was then able to invoke that script.

Later, I got another failure:

```shell
mspreitz@mjs13 perf-tests % cd $CL2_DIR
mspreitz@mjs13 perf-tests % go run cmd/clusterloader.go --testconfig=./testing/load/performance-test-config.yaml --kubeconfig=$HOME/.kube/config --provider=ks --v=2
stat cmd/clusterloader.go: no such file or directory
mspreitz@mjs13 perf-tests % ls cmd
ls: cmd: No such file or directory
```